pom.xml

```diff
@@ -69,7 +69,7 @@
   </pluginRepositories>
 
   <properties>
-    <verapdf.model.syntax.version>[1.0.0,2.0.0)</verapdf.model.syntax.version>
+    <verapdf.model.syntax.version>[1.16.0,1.17.0)</verapdf.model.syntax.version>
```

Comment:
So this is already in integration: https://github.com/veraPDF/veraPDF-model/blob/integration/pom.xml#L72 and should be pulled from there instead. In principle this branch needs to be kept up to date with integration with regular PRs from integration -> pdfua. This kind of commit will lead to conflict in time.